RemoveNAindv.GCL.R

```diff
@@ -1,47 +1,54 @@
 RemoveNAindv.GCL <- function(sillyvec) {
-#~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-#  This function removes any individuals from a SILLY that were not genotyped 
-#  for all markers in markersuite.
-#
-#  LOKI2R.GCL pulls all individuals from a SILLY genotyped for any loci 
-#  existing in LocusControl$locusnames. Sometimes we have individuals that were
-#  only genotyped for a subset of loci in our markersuite. This function
-#  removes those individuals from SILLYs
-#
-#  Argument(s):  
-#  sillyvec <- character vector of SILLYs
-#
-#  Output:
-#  1) this function will modify silly.gcls in your environment to remove NA fish
-#  2) output is a list of FK_FISH_ID of fish removed for each SILLY
-#
-#  Written by Kyle Shedd 10/16/15  
-#~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+  
+  #~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+  #   This function removes any individuals from a SILLY.gcl object that were not genotyped
+  #   for all markers in markersuite.
+  #
+  # Inputs~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+  #   
+  #   sillyvec - a character vector of silly codes (e.g. sillyvec <- c("KQUART06","KQUART08","KQUART09")) 
+  # 
+  # Outputs~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+  #    1) this function will modify silly.gcls in your environment to remove NA fish
+  #    2) output is tibble of the fish IDs removed by silly
+  #
+  # Example~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+  #   load("V:/Analysis/2_Central/Chinook/Cook Inlet/2019/2019_UCI_Chinook_baseline_hap_data/2019_UCI_Chinook_baseline_hap_data.RData")
+  # 
+  #   removedInd <- RemoveNAindv.GCL(sillyvec = sillyvec157)
```

Comment:
This example doesn't really work, we should find an example that has results (i.e. individuals removed), also all of the .gcl objects and LocusControl are in the old list format, so we should add a complete, reproducible example.

Reply:
I forgot to change the workspace image to one that has the new style object, I'll change the directory.  As for being reproducible, should our example data be stored on GitHub somehow or maybe as a data object in our GCL package so the example will work even if you aren't on the dfg network?

Reply:
I've rolled the remove NA individual process in to LOKI2R, I think we can probably get rid of this function now, what do you think?

Reply:
Actually, on second thought, maybe not. If someone converts an old *.gcl list object to the new format, they may still need to remove NA individuals. This function just isn't necessary for dealing with new pulls via LOKI2R